tests/forms_tests/tests/test_forms.py

```diff
@@ -1671,6 +1671,21 @@ def add_prefix(self, field_name):
         self.assertEqual(p.cleaned_data['last_name'], 'Lennon')
         self.assertEqual(p.cleaned_data['birthday'], datetime.date(1940, 10, 9))
 
+    def test_class_prefix(self):
+        # Prefix can be also specified at the class level.
+        class Person(Form):
+            prefix = 'foo'
+
+            first_name = CharField()
```

Comment:
only need 1 field really, right? I'd tend to always put the fields first when defining the class.